apps/indexer-coordinator/src/core/docker.service.ts

```diff
@@ -143,6 +143,27 @@ export class DockerService {
     }
   }
 
+  async getMounts(container: string): Promise<
```

Comment:
You don't have to get mounts of docker container, we already knew the path of the `monitor` folder.

Reply:
`getComposeFileDirectory(cid: string)` can return parent folder of `monitor`